integration-tests/serverless/test-project/.env.yaml

```diff
@@ -0,0 +1,3 @@
+NODE_OPTIONS: "-r dd-trace/init"
+DD_TRACE_DEBUG: "true"
+DD_MINI_AGENT_PATH: "/workspace/node_modules/@thedavl/david-test-datadog-sma/datadog-serverless-agent-linux-amd64/datadog-serverless-trace-mini-agent"
```

Comment:
switch over `@thedavl/david-test-datadog-sma` to the official @datadog/sma package once it's released. We're using an intentionally obscure package name (`sma - serverless mini agent`) for the beta, so customers can't find it easily and we can better control who uses it.